src/app/applications/applications.controller.upload.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+    'use strict';
+
+    // Declare the controllers first
+    angular
+        .module( 'roboconf.applications' )
+        .controller( 'ApplicationsUploadController', applicationsUploadController );
+
+    // Declare the controller functions then.
+    // Specify the injection to prevent errors on minification.
+    applicationsUploadController.$inject = [ 'Restangular', '$scope', 'rAppTemplates' ];
+    
+    // Then comes the function...
+    function applicationsUploadController( Restangular, $scope, rAppTemplates ) {
```

Comment:
Restangular  is not used in your controller, I think you can remove this injection